### PR TITLE
Lxterminal install

### PR DESCRIPTION
**Pull request by CleanMachine1**

### CODE REVIEW

**Comment by CleanMachine1** [comment]
LXterminal is assumed installed in the .py script, however some people might not be able to run, since some people don't use LXDE on pi, like me

**Comment by ghost** [comment]
@CleanMachine1 so does this detect what terminal the user has?

**Comment by CleanMachine1** [comment]
Have you not read the changes?

**Comment by ghost** [comment]
Oh never mind, it installs the lxterminal package.

File: install [hunk 1]
```diff
@@ -10,6 +10,8 @@ Icon=$HOME/pi-ware/icon.png
 Categories=Utility;
 Type=Application
 Terminal=false" > $HOME/.local/share/applications/pi-ware.desktop
+echo "Installing LXTerminal"
+sudo apt install lxterminal
```
Comment (Itai-Nelken) [inline install:14]: Add `-y` to the line that installs lxterminal so apt doesn't prompt for confirmation.

Comment (ghost) [reply install:14]: Thanks! I’ll do that now. Feel free to request an app in an issue.

Comment (ghost) [reply install:14]: So sudo apt install lxterminal -y?

Comment (CleanMachine1) [reply install:14]: yes

**Pull request merged**